Applications/Spreadsheet/Position.h

```diff
@@ -44,6 +45,16 @@ struct Position {
     {
         return !(other == *this);
     }
+
+    URL to_uri() const
```

Comment:
You are using URI and URL somewhat interchangeably here. Let's settle on one, how about just URL?

Reply:
I just can't see a URL with a custom protocol and scheme as a URL, so I tend to think of it as a URI...
I'll swap over to URL.